public/app/plugins/datasource/opentsdb/components/TagSection.tsx

```diff
@@ -191,12 +199,12 @@ export function TagSection({
               </button>
             </div>
           </div>
-        </div>
+        </Stack>
       )}
-      <div className="gf-form gf-form--grow">
-        <div className="gf-form-label gf-form-label--grow"></div>
-      </div>
-    </div>
+      <Stack gap={0} grow={1}>
+        <InlineLabel> </InlineLabel>
+      </Stack>
+    </Stack>
```

Comment:
Wonder if this empty element could be avoided?

Reply:
um no I don't think, this gives the grey look at the end

Reply:
also, InlinFormLabel and InlineLabel have same functionality, so which is preferred? InlineFormLabel is not a storybook component here https://developers.grafana.com/ui/canary/index.html?path=/docs/forms-inlinelabel--docs. LMK